pythoncalculator/multiply.py

```diff
@@ -0,0 +1,2 @@
+def multiply(x, y):
+    return x + y
```

Comment:
I think you might have meant to do * not + 

```suggestion
    return x * y
```